ceph: Update pg_num related documents (#5247)
rook/rook: Documentation/ceph-configuration.md

      enabled: true
```

In Octopus (v15.2.x), this module is enabled by default without the above-mentioned setting.

With that setting, the autoscaler will be enabled for all new pools. If you do not desire to have
the autoscaler enabled for all new pools, you will need to use the Rook toolbox to enable the module
and [enable the autoscaling](https://docs.ceph.com/docs/master/rados/operations/placement-groups/)
on individual pools.

The autoscaler is not enabled for the existing pools after enabling the module. So if you want to

> enable the autoscaling for these pools, you should do so by yourself.

Reviewer: ```suggestion
enable the autoscaling for these existing pools, they must be configured from the toolbox.
```

Author: Your suggestion makes sense. Fixed.
